README.md

```diff
@@ -86,15 +86,17 @@ It's best to have at least 30GB of free space for Docker containers and images.
 
 ## Releasing a new version
 
-Travis is setup to automatically publish tagged releases to the npm registry. So if you want to release a new version of YamUI, just do the following:
+Once you have updated `package.json` to the new version you wish to release, you may do the following:-
 
 ```sh
-git checkout -b awesome-release-branch
-npm version patch -m "Release version %s"
-git push
```

Comment:
All the stuff about bumping version in a separate branch still applies.

Reply:
ya we should add this new info after the `npm version` info